install/index.php

```diff
@@ -1847,7 +1845,7 @@ function revertSetting(defval, inpid)
 
 		if(substr($cookiedomain, 0, 4) == "www.")
 		{
-			$cookiedomain = my_substr($cookiedomain, 4);
+			$cookiedomain = substr($cookiedomain, 4);
```

Comment:
But the first 4 removed chars are always `www.` because of the conditional above. Doesn't matter if the rest of the domain contains special characters - they won't be affected by the `substr()` at all.